pyproject.toml

```diff
@@ -8,7 +8,7 @@ build-backend = "setuptools.build_meta"
 [project]
 name = "ops-scenario"
 
-version = "6.0.3"
+version = "7.0.0"
```

Comment:
I think it's time to add @tonyandrewmeyer to `authors` two lines down 😄

Reply:
I think before 7.0 is released, we could put Charm-Tech in as maintainers, or maybe to authors. I'd like the email address to be working...